libbeat/docs/release-notes/breaking/breaking-7.6.asciidoc

```diff
@@ -27,3 +27,20 @@ setup.ilm.policy_name: "%{[agent.name]}-%{[agent.version]}"
 ----
 
 // end::notable-breaking-changes[]
```

Comment:
The notatable-breaking-changes opening/closing tags need to wrap the whole block of changes, not each change. (I see why you did this, though, because the comment at the top is a little misleading.)

So delete lines 29 and 31.